### PR TITLE
Story #575 combine prepare topics

### PR DESCRIPTION
**Pull request by shashi165**
### Changes for [story #575](https://github.com/mojaloop/project/issues/575)
1. Combine the transfer prepare topics
2. Remove the Kafka cron-job to register DFSP Specific Prepare Handlers

### Tests

```
=============================== Coverage summary ===============================
Statements   : 100% ( 3068/3068 )
Branches     : 100% ( 755/755 )
Functions    : 100% ( 428/428 )
Lines        : 100% ( 3039/3039 )
================================================================================

1..1111
# tests 1111
# pass  1111

# ok

```

### CODE REVIEW

**Comment by rmothilal** [comment]
@shashi165 i believe that you are missing out some changes. for example you need to make changes to src/handlers/index.js please look at the changes in the performance branch and copy those.

File: src/shared/setup.js [hunk 1]
```diff
@@ -146,7 +146,7 @@ const createHandlers = async (handlers) => {
       Logger.info(`Handler Setup - Registering ${JSON.stringify(handler)}!`)
       switch (handler.type) {
         case 'prepare':
-          await RegisterHandlers.transfers.registerPrepareHandlers(handler.fspList)
+          await RegisterHandlers.transfers.registerPrepareHandler()
           if (!Config.HANDLERS_CRON_DISABLED) {
```
Comment (rmothilal) [inline src/shared/setup.js]: this condition should be removed as it is no longer applicable as there is a single topic created

File: package.json [hunk 2]
```diff
@@ -21,7 +21,7 @@
   },
   "scripts": {
     "start": "run-p start:api start:admin",
-    "start:api": "CLEDG_HOSTNAME=http://localhost:3000 node src/api/index.js",
+    "start:api": "CLEDG_HOSTNAME=http://localhost:3000 node --nolazy --inspect-brk=9230 src/api/index.js",
```
Comment (mdebarros) [inline package.json]: Get rid of the --nolazy --inspect parameters please

**Comment by mdebarros** [comment]
I am happy with this. @rmothilal please review and confirm you are happy.

File: src/handlers/api/admin/routes.js [hunk 3]
```diff
@@ -83,7 +83,7 @@ module.exports = [
   {
```
[the remark this answers is not in the document]
Comment (shashi165) [reply src/handlers/api/admin/routes.js]: done

**Pull request merged**